docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,78 @@
+# ADR 14: Proportional Slashing
+
+## Changelog
+
+- 2019-10-15: Initial draft
+
+## Context
+
+In Proof of Stake-based chains, centralization of consensus power amongst a small set of validators causes harm to the network due to increased risk of censorship, liveness failure, fork attacks, etc.  However, while this centralization causes a negative externality to the network, it is not directly felt by the stakers contributing towards delegating towards already large validators.  We would like a way to pass on the negative externality cost of centralization onto those large validators and their stakers.
+
+## Decision
+
+### Design
+
+To solve this problem, we will implement a procedure called Proportional Slashing.  The desire is that the larger a validator is, the more they should be slashed.  The first naive attempt is to make a validator's slash percent proportional to their share of consensus voting power.
+
+```
+slash_amount = k * power // where k is a constant and power is the faulting validator's voting power.
+```
+
+However, this will incentivize validators with large amounts of stake to split up their voting power amongst accounts, so that if they fault, they all get slashed at a lower percent.  The solution to this is to take into account not just a validator's own voting percentage, but also the voting percentage of all the other validators who slash in a short time window.
+
+```
+slash_amount = k * (power_1 + power_2 + ... + power_n) // where power_i is the voting power of the ith validator faulting in the period
+```
+
+Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same window, they both will still get slashed at the sum 10% amount.
```

Comment:
It's not clear to me if a validator that splits up its power amongst numerous accounts should deserve to have its slash fraction potentially include faults of other validators who are not involved in the fault at hand.

Reply:
Also, is this different from the naive solution? If a validator that deserves to be slashed 10% splits up their power into two accounts, they'll still be slashed 10% cumulatively if both accounts commit the same infractions, no?

Reply:
It is the responsibility of validators to decorrelate themselves from other validators.  The more validators that fault at once, even accidentally, causes harm to the network because it decreases the resources that an attacker needs to acquire to attack the network.  Thus it is logical to punish correlated faults, even if accidental.

Reply:
Also, no it's different than the naive solution because in the naive solution, both validators of 5% power will each only get slashed at 5%.  Thus, the total 10% amongst two validators is still only getting slashed at 5%.

We want the total 10% to get slashed at a rate >= 10%.

Reply:
Ok, I see how this is better than the naive solution. However, `It is the responsibility of validators to decorrelate themselves from other validators.` doesn't much make sense to me but I get the gist of what this is trying to accomplish.